packages/nextjs-mf/src/plugins/LoadDependenciesTemplate.ts

```diff
@@ -0,0 +1,45 @@
+//@ts-ignore
+import Template from '../../utils/Template';
+
+export const loadDependenciesTemplate = (RuntimeGlobals: any) => {
+  return Template.asString([
+    'function loadDependencies(libKeys, cnn) {',
```

Comment:
This is what loads the eager modules. Webpack queues initial chunks till this resolves.